src/backend/main/py.main/caliopen_main/parsers/vcard.py

```diff
@@ -193,6 +193,18 @@ def parse_vcard(vcard):
                                 ke.type = j
                     ke.name = ('key{}{}'.format(ke.type,ke.size))
                     new_contact.public_keys.append(ke)
+
+        elif v == 'tel':
+            for tel in vcard.contents['tel']:
+                phone = NewPhone()
+                phone.is_primary = False
+                try:
+                    func = PhoneNumberType.validate_phone
+                    phone.number = func(PhoneNumberType(),tel.value)
+                    new_contact.phones.append(phone)
+                except:
+                    pass
```

Comment:
For the moment invalid phone number (not international format) will be dropped.

We need to define a strategy to guess a country related to a phone number for the normalization library we used able to process.

We can relax condition and store phone number even if it can't be normalized but that's not goal for long term